tests/unit/backend/layers/business/test_business.py

```diff
@@ -1475,5 +1475,32 @@ def test_dataset_published_at_with_replaced_dataset_ok(self):
         self.assertEqual(dataset_1.canonical_dataset.published_at, published_collection.published_at)
 
 
+class TestCollectionUtilities(BaseBusinessLogicTestCase):
+    def test__delete_datasets_from_public_access_bucket(self):
```

Comment:
add a docstring. You are testing delete published datasets and unpublished datasets, add those details here.